11-networking-in-flutter/projects/starter/pubspec.yaml

```diff
@@ -12,12 +12,11 @@ dependencies:
   flutter:
     sdk: flutter
 
-  path: ^1.8.1
```

Comment:
I remember putting this in because I needed it. Why is it being removed?

Reply:
It's not described in the chapter. Plus it’s a transient dependency of some other package, so it’s included anyway. If it's needed in subsequent chapters we’ll explicitly tell the reader to add it.